lighthouse-core/config/default-config.js

```diff
@@ -172,8 +172,6 @@ const defaultConfig = {
     loadFailureMode: 'ignore',
     gatherers: [
       'service-worker',
```

Comment:
should `service-worker` move to `redirectPass` and only have two passes? `service-worker` would fail if the site doesn't redirect, but should `http-redirect` move to use `driver.fetcher` anyways?

Reply:
> should `service-worker` move to `redirectPass` and only have two passes? `service-worker` would fail if the site doesn't redirect, but should `http-redirect` move to use `driver.fetcher` anyways?

and actually, with the new DevTools offline check, which is doing a real request, that means the ServiceWorker has to be activated and controlling the page. Presumably that means `getServiceWorkerVersions`/`getServiceWorkerRegistrations` will return the info, so do we even need a second pass?

Reply:
i love it. filed https://github.com/GoogleChrome/lighthouse/issues/11822